### DISCUSSION TITLE
Default dates in Date picker incorrect

### DISCUSSION THREAD

**Discussion by developer-uhwi** [Budibase/budibase#9427] [Help]
I am trying to set the default date in the picker, I am using the following value returned from a JSON field "2023-02-28 00:00:00.000Z" instead of showing "February 28, 2023" it shows "February 27, 2023".

If I used the following formats
{{ date  "2023-02-28 00:00:00.000Z" "YYYY-MM-DD" "UTC" }} = February 27, 2023
{{ date  "2023-02-28 00:00:00.000Z" "YYYY-MM-DD" "America/Bogota" }}  = February 26, 2023
{{ date  "2023-02-28 00:00:00.000Z" "YYYY-MM-DD" }}  = February 27, 2023
Setting default value to "2023-02-28 00:00:00.000Z" = February 27, 2023

What to do????

**Comment by melohagan** [comment]
Hey @developer-uhwi 

I think in the longer term we're looking to move away from the current Date Picker library we use (flatpickr).

In the meantime you could try using a custom date picker that may have better locale support: https://github.com/melohagan/budibase-component-kasper-date-picker

Otherwise you could raise a bug issue, following the bug template, but it may be a while before it gets looked at.

**Comment by developer-uhwi** [comment]
I am trying the date plugin you have recommended, but unable to set the
default date from a variable.

**Reply by melohagan** [reply]
That's unfortunate - feel free to raise an issue on that date picker repo and I'll take a look when I get a chance.
